examples/pytorch_sample/build_and_push.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
```

Comment:
@johnugeorge Can we use the current build process for this image? Right now we are using the gcloud container builder to build the pytorch-operator container and we can use that for this Dockerfile as well https://github.com/kubeflow/pytorch-operator/blob/master/scripts/build.sh#L40

That way we can test any changes to this smoke test in our CI system

Reply:
Something like what is being done in this new PR https://github.com/kubeflow/pytorch-operator/pull/38/files